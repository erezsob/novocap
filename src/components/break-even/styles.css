.container {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 20px;
    width: 90%;
    padding-top: 20px;
    color: white;
    font-weight: bold;
    font-size: 20px;
}

.barContainer {
    margin: 20px;
    width: 150px;
    height: 150px;
    position: relative;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.title {
    display: flex;
    flex-direction: column;
    padding-top: 13px;
}

.date {
    font-size: 11px;
    font-weight: 300;
}

.addCosts {
    color: #fff;
    font-size: 19px;
}