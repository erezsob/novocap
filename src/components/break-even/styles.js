import { blueMain } from '../../common/variables.css';

export default {
  card: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    boxShadow: 'none',
    height: 208,
    backgroundColor: blueMain,
    cursor: 'pointer'
  },
  iconButton: {
    color: 'white',
    width: 24
  }
};
