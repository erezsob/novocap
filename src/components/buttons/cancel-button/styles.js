import { pinkyRed } from '../../../common/variables.css';

export default {
  cancel: {
    color: pinkyRed
  }
};
