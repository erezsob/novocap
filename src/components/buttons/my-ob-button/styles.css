@value blueLight, bpXSmall from '../../../common/variables.css';

.button {
    border-radius: 4px;
    background-color: blueLight;
    color: white;
    padding: 8px 0;
    font-size: 12px;
    display: table;
    border: 0;
    outline: 0;
    cursor: pointer;
}

.bigger {
    padding: 10px 0;
    font-size: 16px;
}

.icon,
.text {
    vertical-align: middle;
    text-align: center;
}

.text {
    padding: 0 20px;
    font-weight: lighter;
}

.icon ~ .text {
    display: none;
}

.icon {
    display: table-cell;
    padding: 0 10px;
    position: relative;
    top: 2px;
}

@media (min-width: bpXSmall) and (min-device-width: bpXSmall) {
    .icon ~ .text {
        padding-left: 0;
        display: table-cell;
    }
}
