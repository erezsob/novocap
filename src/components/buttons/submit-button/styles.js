import { greenGrass } from '../../../common/variables.css';

export default {
  submit: {
    color: greenGrass
  }
};
