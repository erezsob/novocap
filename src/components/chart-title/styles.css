.label {
    letter-spacing: 0.05em;
    font-size: 20px;
    border-bottom: 1px solid #efefef;
    padding: 20px;
    margin-bottom: 20px;
    font-weight: bold;
    color: #323435;
}