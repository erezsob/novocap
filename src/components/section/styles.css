.container {
    margin: 30px auto;
    box-sizing: border-box;
    width: 90%;
}
