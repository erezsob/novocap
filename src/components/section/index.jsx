// @flow
import * as React from 'react';
import Grid from 'material-ui/Grid';
import { SectionTitle } from './../';
import styles from './styles.css';

type Props = {
  title: string,
  children: React.Node
};

function Section({ title, children }: Props) {
  return (
    <div className={styles.container}>
      <Grid container spacing={24} justify={'center'}>
        <Grid item xs={12}>
          <SectionTitle title={title} />
        </Grid>
        {children}
      </Grid>
    </div>
  );
}

export default Section;
