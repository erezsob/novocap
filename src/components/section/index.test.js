import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Section from './';
import { SectionTitle } from '../../components';
import styles from './styles.css';

describe('<Section />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Section title="asdf" />);
  });

  it('renders', () => {
    expect(wrapper).toHaveLength(1);
  });

  it('renders with the right css class', () => {
    expect(wrapper.hasClass(`${styles.container}`)).toBeTruthy();
  });

  it('renders a <SectionTitle />', () => {
    expect(wrapper.find(SectionTitle)).toHaveLength(1);
  });

  it('renders properly', () => {
    const tree = renderer
      .create(
        <Section>
          <dummy-children>Dummy Child</dummy-children>
          <dummy-children>Dummy Child</dummy-children>
        </Section>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
