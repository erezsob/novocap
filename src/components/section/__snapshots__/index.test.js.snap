// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> renders properly 1`] = `
<div
  className="container"
>
  <div
    className="MuiGrid-typeContainer-1 MuiGrid-spacing-xs-24-24 MuiGrid-justify-xs-center-18"
  >
    <div
      className="MuiGrid-typeItem-2 MuiGrid-grid-xs-12-38"
    >
      <div
        className="title"
      />
    </div>
    <dummy-children>
      Dummy Child
    </dummy-children>
    <dummy-children>
      Dummy Child
    </dummy-children>
  </div>
</div>
`;
